src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -238,6 +238,26 @@ def load_arguments(self, _):
     with self.argument_context('aks remove-dev-spaces') as c:
         c.argument('prompt', options_list=['--yes', '-y'], action='store_true', help='Do not prompt for confirmation')
 
+    # OpenShift command argument configuration
+    with self.argument_context('openshift') as c:
+        c.argument('resource_name', name_type, help='Name of the managed OpenShift cluster.',
+                   completer=get_resource_name_completion_list('Microsoft.ContainerService/OpenShiftManagedClusters'))
+        c.argument('name', name_type, help='Name of the managed OpenShift cluster.',
+                   completer=get_resource_name_completion_list('Microsoft.ContainerService/OpenShiftManagedClusters'))
+        c.argument('compute_count', options_list=['--compute-count', '-c'], type=int, default=4)
+        c.argument('tags', tags_type)
+
+    with self.argument_context('openshift create') as c:
+        c.argument('name', validator=validate_linux_host_name)
+        c.argument('compute_vm_size', options_list=['--compute-vm-size', '-s'])
+        c.argument('fqdn', required=True)
```

Comment:
This shouldn't be necessary. If it doesn't have a default value in the command signature, the CLI will know it is required.

Reply:
Should I remove the whole line (253) or just the `required=true` ?

Reply:
You can remove the whole line.